### ISSUE TITLE
Add Pointwise ops to PyTorch Frontend

### ISSUE THREAD

**Issue by djl11** [unifyai/ivy#12]
Add [Pointwise ops](https://pytorch.org/docs/stable/torch.html#pointwise-ops) to PyTorch frontend:

- [ ] abs
- [ ] absolute
- [ ] acos
- [ ] arccos
- [ ] acosh
- [ ] arccosh
- [ ] add
- [ ] addcdiv
- [ ] addcmul
- [ ] angle
- [ ] asin
- [ ] arcsin
- [ ] asinh
- [ ] arcsinh
- [ ] atan
- [ ] arctan
- [ ] atanh
- [ ] arctanh
- [ ] atan2
- [ ] bitwise_not
- [ ] bitwise_and
- [ ] bitwise_or
- [ ] bitwise_xor
- [ ] bitwise_left_shift
- [ ] bitwise_right_shift
- [ ] ceil
- [ ] clamp
- [ ] clip
- [ ] conj_physical
- [ ] copysign
- [ ] cos
- [ ] cosh
- [ ] deg2rad
- [ ] div
- [ ] divide
- [ ] digamma
- [ ] erf
- [ ] erfc
- [ ] erfinv
- [ ] exp
- [ ] exp2
- [ ] expm1
- [ ] fake_quantize_per_channel_affine
- [ ] fake_quantize_per_tensor_affine
- [ ] fix
- [ ] float_power
- [ ] floor
- [ ] floor_divide
- [ ] fmod
- [ ] frac
- [ ] frexp
- [ ] gradient
- [ ] imag
- [ ] ldexp
- [ ] lerp
- [ ] lgamma
- [ ] log
- [ ] log10
- [ ] log1p
- [ ] log2
- [ ] logaddexp
- [ ] logaddexp2
- [ ] logical_and
- [ ] logical_not
- [ ] logical_or
- [ ] logical_xor
- [ ] logit
- [ ] hypot
- [ ] i0
- [ ] igamma
- [ ] igammac
- [ ] mul
- [ ] multiply
- [ ] mvlgamma
- [ ] nan_to_num
- [ ] neg
- [ ] negative
- [ ] nextafter
- [ ] polygamma
- [ ] positive
- [ ] pow
- [ ] quantized_batch_norm
- [ ] quantized_max_pool1d
- [ ] quantized_max_pool2d
- [ ] rad2deg
- [ ] real
- [ ] reciprocal
- [ ] remainder
- [ ] round
- [ ] rsqrt
- [ ] sigmoid
- [ ] sign
- [ ] sgn
- [ ] signbit
- [ ] sin
- [ ] sinc
- [ ] sinh
- [ ] sqrt
- [ ] square
- [ ] sub
- [ ] subtract
- [ ] tan
- [ ] tanh
- [ ] true_divide
- [ ] trunc
- [ ] xlogy

**Comment by soran-ghaderi** [comment]
[ ] #1643

**Comment by Amrabdelhamed611** [comment]
#2247

**Comment by pritam1322** [comment]
cos #2467

**Comment by pritam1322** [comment]
sin #2533
acos #2535

**Comment by Winvoker** [comment]
abs #2538

**Comment by gary30404** [comment]
- [ ] #2582

**Comment by mohamedelabbas1996** [comment]
#2673

**Comment by AbdallahOmarAhmed** [comment]
acosh #2730

**Comment by AbdallahOmarAhmed** [comment]
- [ ] #2730

**Comment by AmelxJamal** [comment]
#2735

**Comment by kunwarsaaim** [comment]
- [ ] #2739

**Comment by kunwarsaaim** [comment]
- [ ] #2740

**Comment by lightspeedana** [comment]
- [ ] #2782

**Comment by kunwarsaaim** [comment]
- [ ] #2878

**Comment by ytg32** [comment]
[ ] #2933

**Comment by ytg32** [comment]
[] #2933

**Comment by ytg32** [comment]
#2933

**Comment by ytg32** [comment]
- [ ] #2933

**Comment by kunwarsaaim** [comment]
- [ ] #2993

**Comment by kunwarsaaim** [comment]
- [ ] #2995

**Comment by guillesanbri** [comment]
- [ ] #3014

**Comment by hirwa-nshuti** [comment]
square #3033

**Comment by kunwarsaaim** [comment]
- [ ] #3035

**Comment by Emperor-WS** [comment]
#3058

**Comment by Mohammad-Talaat7** [comment]
#3061

**Comment by ream-kh** [comment]
- [ ] #3135

**Comment by kunwarsaaim** [comment]
- [ ] #3163

**Comment by maazzzzz** [comment]
- [ ] #3167

**Comment by ahmedelmogtaba** [comment]
log2 #3183

**Comment by kunwarsaaim** [comment]
- [ ]  #3198

**Comment by Emperor-WS** [comment]
- [ ] #3200

**Comment by Mohammad-Talaat7** [comment]
- [ ] #3204

**Comment by EKostinoudis** [comment]
- [ ] #3374

**Comment by primetimepaper** [comment]
pow #3458

**Comment by vincycode7** [comment]
- [ ] #3948

**Comment by vincycode7** [comment]
[Added absolute function and test to Pytorch Frontend](https://github.com/unifyai/ivy/pull/3951)

**Comment by daniabib** [comment]
#4268

**Comment by aditya-hari** [comment]
- [ ] #5354

**Comment by aditya-hari** [comment]
- [ ] #5355

**Comment by aditya-hari** [comment]
- [ ] multiply

**Comment by hirwa-nshuti** [comment]
- [ ] #6525

**Comment by hirwa-nshuti** [comment]
- [ ] #6526

**Comment by hirwa-nshuti** [comment]
- [ ]  #6527

**Comment by hirwa-nshuti** [comment]
- [ ] #6528

**Comment by hirwa-nshuti** [comment]
- [ ] #6529

**Comment by zhengyaogu** [comment]
#6511 This merge is not reflected in the list above.

**Comment by hirwa-nshuti** [comment]
- [ ] #6730

**Comment by zhengyaogu** [comment]
pow #6891

**Comment by AnnaTz** [comment]
> #6511 This merge is not reflected in the list above.

Thank you! I have added it to the list.

**Comment by EJKonge** [comment]
- [ ] #7157

**Comment by usama-baloch** [comment]
#7285

**Comment by usama-baloch** [comment]
cosh function is already implemented by someone, kindly update it on ToDo List. Here is the implementation Link: https://github.com/unifyai/ivy/blob/master/ivy/functional/frontends/torch/pointwise_ops.py

**Comment by usama-baloch** [comment]
sigmoid #7298

**Comment by usama-baloch** [comment]
log #7300

**Comment by modanesh** [comment]
- [ ] #7685

**Comment by ReneFabricius** [comment]
- [] #7752

**Comment by ReneFabricius** [comment]
pow is open, but already implemented by @zhengyaogu

**Comment by AnnaTz** [comment]
> pow is open, but already implemented by @zhengyaogu

Thank you. I will make sure to note it as completed..

**Comment by ReneFabricius** [comment]
- [] #7757

**Comment by usama-baloch** [comment]
Hi @AnnaTz.
log is open, but it is already implemented by me, can you mark it as completed?

**Comment by AnnaTz** [comment]
> Hi @AnnaTz. log is open, but it is already implemented by me, can you mark it as completed?

Thank you! Noted.

**Comment by aparna2071** [comment]
- [ ] #10683

**Comment by codesankalp** [comment]
#10711

**Comment by codesankalp** [comment]
- [ ] #10712

**Comment by KevinUnadkat** [comment]
#10808

**Comment by KevinUnadkat** [comment]
#10867

**Comment by KevinUnadkat** [comment]
#11025

**Comment by KevinUnadkat** [comment]
#11026

**Comment by usama-baloch** [comment]
#11030

**Comment by AnnaTz** [comment]
- [ ] #10867

**Comment by AnnaTz** [comment]
- [ ] #11026

**Comment by aparna2071** [comment]
Hi @AnnaTz lerp is open but is implemented by me, can you please mark it as complete

**Comment by KevinUnadkat** [comment]
#11104

**Comment by KevinUnadkat** [comment]
#11120

**Comment by AnnaTz** [comment]
- [ ] #11120

**Comment by princeminz** [comment]
- [ ] #13099

**Comment by princeminz** [comment]
- [ ] #13562

**Comment by NotEnoughSnow** [comment]
- [ ] #13763

**Comment by abdokaseb** [comment]
#13846

**Comment by Dhruv2012** [comment]
- [ ] #13895

**Comment by Dhruv2012** [comment]
- [ ] #13952

**Comment by binh234** [comment]
- [] #14002

**Comment by binh234** [comment]
- [] #14033

**Comment by binh234** [comment]
- [] #14034

**Comment by Dhruv2012** [comment]
- [ ] #14040

**Comment by binh234** [comment]
- [] #14092

**Comment by binh234** [comment]
- [] #14094

**Comment by binh234** [comment]
- [] #14096

**Comment by binh234** [comment]
- [] #14110

**Comment by tdjogi010** [comment]
- [ ] #14169

**Comment by NotEnoughSnow** [comment]
- [ ] #13763

**Comment by NotEnoughSnow** [comment]
- #14452

**Comment by Lucas-Strange** [comment]
- [ ] #15517

**Comment by NotEnoughSnow** [comment]
- #14958

**Comment by AniTho** [comment]
#19215

**Comment by bbyiringiro** [comment]
- [ ] #19281

**Comment by Krishna-Boppana** [comment]
#21008

**Comment by lukaM19** [comment]
#21758

**Comment by Sjsingh101** [comment]
- [ ] #21874

**Comment by Abdullah-Meda** [comment]
- [ ] #22167

**Comment by pangeran-bottor** [comment]
* [ ]  [quantized_batch_norm #22260](https://github.com/unifyai/ivy/issues/22260)

**Comment by pangeran-bottor** [comment]
- [ ] #22260

**Comment by rajendrabaskota** [comment]
#22486

**Comment by mcyhx** [comment]
- [ ] #26765
